content/geoip/geolocate-an-ip/databases.mdx

```diff
@@ -1,18 +1,249 @@
+| Rust                 | maxminddb               | [Cargo](https://crates.io/crates/maxminddb)                           | [API docs](https://oschwald.github.io/maxminddb-rust/maxminddb/struct.Reader.html) | [GitHub](https://github.com/oschwald/maxminddb-rust)         |
+| Scala                | maxmind-geoip2-scala    |                                                                       | [README](https://github.com/Sanoma-CDA/maxmind-geoip2-scala/blob/master/README.md) | [GitHub](https://github.com/Sanoma-CDA/maxmind-geoip2-scala) |
+| Swift (C extension)  | MMDB-Swift              | [MMDB-Swift](https://cocoapods.org/pods/MMDB-Swift)                   | [CocoaPods](hhttps://cocoapods.org/pods/MMDB-Swift)                                | [GitHub](https://github.com/lexrus/MMDB-Swift)               |
 
 ## Integrations
 
-(todo)
+<Alert type="warning">
+  Maxmind does **not** offer support for these integrations. Please contact the
+integrator for assistance.
+
+Use at your own risk.
+</Alert>
+
+| Application                    | Platform                | Link                                                                                                        |
+| ------------------------------ | ----------------------- | ----------------------------------------------------------------------------------------------------------- |
+| Ad-serving                     | Revive Adserver         | [Geotargeting with Revive Adserver](https://www.revive-adserver.com/faq/geotargeting-with-revive-adserver/) |
+| Analytics                      | Piwik/Matomo            | [GEO LOCATE YOUR VISITORS](https://matomo.org/docs/geo-locate/)                                             |
+| Analytics                      | Snowplow Analytics      | [IP lookups enrichment](https://github.com/snowplow/snowplow/wiki/IP-lookups-enrichment)                    |
+| Content Delivery               | Varnish Software        | [libvmod-geoip2](https://github.com/fgsch/libvmod-geoip2)                                                   |
+| E-commerce                     | WooCommerce / WordPress | [MaxMind Geolocation Integration](https://docs.woocommerce.com/document/maxmind-geolocation-integration/)   |
+| Load balancer                  | NGINX                   | [ngx_http_geoip2_module](https://github.com/leev/ngx_http_geoip2_module)                                    |
+| Log Analyzer / Data Processing | Logstash                | [Geoip filter plugin](https://www.elastic.co/guide/en/logstash/current/plugins-filters-geoip.html)          |
+| Network Protocol Analyzer      | Wireshark               | [How To Use GeoIP With Wireshark](https://wiki.wireshark.org/HowToUseGeoIP)                                 |
```

Comment:
Would https://www.elastic.co/guide/en/elasticsearch/reference/current/geoip-processor.html be appropriate to add to this list?